qi-lib/flow/core/deforest.rkt

```diff
@@ -28,6 +28,26 @@
 
 (begin-for-syntax
 
+  ;; Partially reconstructs original flow expressions. The chirality
+  ;; is lost and the form is already normalized at this point though!
```

Comment:
@michaelballantyne Would it be possible to get the source expression as a syntax property on the expanded syntax object? We have this neato (and a bit hacky 😆 ) **de-expander** ✨ 📶 for now, but would be ideal to have the actual thing eventually (no rush). I vaguely seem to recall you talking about something like this a long time ago as one of the benefits of syntax-spec, that it retains the whole context of expansion and not just individual macro applications.